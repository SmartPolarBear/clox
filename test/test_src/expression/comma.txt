R"(
print 1,2,3,4,5;

var a=10;
var b=20;

print a,b,a+b;

fun add(a,b){
    return a+b;
}

print add(1,2),add(3,4);
)"