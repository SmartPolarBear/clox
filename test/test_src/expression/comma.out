R"(5.000000
30.000000
7.000000)"